### PR TITLE
lowdown: update to 0.9.0

### PR DESCRIPTION
**Pull request by judaew**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.6 20G165 x86_64
Xcode 13.0 13A233

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by reneeotten** [comment]
thanks @judaew - can you please fix the `port lint` warnings?

File: textproc/lowdown/Portfile [hunk 1]
```diff
@@ -1,21 +1,24 @@
-PortSystem              1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name                    lowdown
-categories              textproc
-version                 0.3.2
-license                 BSD
+PortSystem          1.0
+PortSystem          1.0
```
Comment (herbygillot) [inline textproc/lowdown/Portfile:4]: `PortSystem` specified twice?

File: textproc/lowdown/Portfile [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+license             BSD
 
-checksums \
-        rmd160 f105294316729f92b0f2dacf5559f49f86a1a677 \
-        sha256 740cf6025c43ceaae9bcc1ffc3e94986aa62d65d91ef8173d444a1977c562094
+description         simple markdown translator
+long_description    lowdown translates markdown to either HTML \
+                    or a roff document using either -man or -ms
+homepage            https://kristaps.bsd.lv/lowdown/
+master_sites        https://kristaps.bsd.lv/lowdown/snapshots/
```
Comment (herbygillot) [inline textproc/lowdown/Portfile:15]: Use `${homepage}` to avoid repetition

File: textproc/lowdown/Portfile [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
-description             simple markdown translator
-long_description        lowdown translates markdown to either HTML \
-                        or a roff document using either -man or -ms
-homepage                http://kristaps.bsd.lv/lowdown/
-master_sites            http://kristaps.bsd.lv/lowdown/snapshots/
-maintainers             nomaintainer
-platforms               darwin
+name                lowdown
+categories          textproc
+version             0.9.0
```
Comment (herbygillot) [inline textproc/lowdown/Portfile:8]: Missing a `revision` field

**Comment by judaew** [comment]
Done

**Pull request merged**